docs/guide/extending_ez_platform.md

```diff
@@ -753,3 +753,148 @@ services:
         arguments:
             $shortDateTimeFormatter: '@ezplatform.user.settings.short_datetime_format.formatter'
 ```
+
+## User settings
+
+You can add new preferences to the User settings menu in the Back Office.
+
+To do so, create a setting class implementing two interfaces:
+`ValueDefinitionInterface` and `FormMapperInterface`.
+
+In this example the class is located in `AppBundle/Setting/Unit.php`
+and enables the user to select their preference for metric or imperial unit systems.
```

Comment:
`his preference`

Reply:
We actually use "their" on purpose to avoid having to go with "his/her" (it's slowly becoming a standard).